test/cpp/jit/test_gpu.cpp

```diff
@@ -2083,6 +2083,155 @@ void testGPU_FusionComputeAtNoCommonConsumer() {
   TORCH_CHECK(at::allclose(kernel_tv6, t6));
 }
 
+namespace {
+void checkConcretized(
```

Comment:
nit: please add an empty line after `namespace {` and one before the closing `}`

Reply:
Updated, thanks!